Packs/Maltiverse/README.md

```diff
@@ -0,0 +1,248 @@
+| Maltiverse.IP.Blacklist.Description | String | Describes the reason for the IP to be in the blacklist | 
+| Maltiverse.IP.Blacklist.Source | String | The name of sources that reported the indicator | 
+| Maltiverse.IP.Tags | String | The type of indicator | 
+
+
+##### Command Example
+```!ip ip=8.8.8.8```
+
+##### Human Readable Output
+
+
+### 2. domain
+---
+Checks the reputation of a Domain
+##### Required Permissions
+**FILL IN REQUIRED PERMISSIONS HERE**
+##### Base Command
+
+`domain`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| domain | Domain address to check | Required | 
+| threshold | If the number of positives is higher than the threshold, the domain will be considered malicious. If the thershold is not specified, the default domain threshold, as configured in the instance settings, will be used. | Optional | 
+| fullResponse | Return all of the results, note that it can be thousands of results. Prefer not to use in playbooks. The default value is "false | Optional | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Domain.Name | String | The domain name | 
+| Domain.CreationDate | Date | Date when a IoC has been inserted for the first time | 
+| Domain.ModificationDate | Date | Date when a IoC has been updated for the last time | 
+| Domain.Tld | Number | Top Level Domain of the hostname | 
+| Domain.ASName | String | Autonumous system name of the domain | 
+| DBotScore.Score | Number | The DBot score | 
+| DBotScore.Type | String | The type of indicator | 
+| DBotScore.Vendor | String | The vendor used to calculate the score | 
+| DBotScore.Indicator | String | The indicator that was tested | 
+| Maltiverse.Domain.Address | String | The domain name | 
+| Maltiverse.Domain.Classification | String | Qualitative maliciousness classification for an IoC. Possible value are malicious, suspicious, neutral and whitlist | 
+| Maltiverse.Domain.Blacklist.Firstseen | Date | First time that the IoC was seen | 
+| Maltiverse.Domain.Blacklist.LastSeen | Date | Last time that the IoC was seen | 
+| Maltiverse.Domain.Blacklist.Description | Unknown | Describes the reason for the domain to be in the blacklist | 
+| Maltiverse.Domain.Blacklist.Source | String | The name of sources that reported the indicator | 
+| Maltiverse.Domain.Tag | String | Attribute to label an IoC | 
+| Maltiverse.Domain.ModificationTime | Date | Date when the IoC was updated for the last time | 
+| Maltiverse.Domain.CreationTime | Date | Date when a IoC was inserted for the first time | 
+| Maltiverse.Domain.TLD | String | Top level domain of the hostname | 
+| Maltiverse.Domain.ResolvedIP.IP | String | Stores an IP that was resolved by the domain | 
+| Maltiverse.Domain.ResolvedIP.Timestamp | Date | Stores an timestamp when an IP address has been resolved by the domain | 
+
+
+##### Command Example
+```!domain domain=google.com```
+
+##### Human Readable Output
+
+
```

Comment:
Need to double-check, but historically we would do a screenshot of what get's returned to the war room.